packages/grid/_modules_/grid/hooks/root/useEvents.ts

```diff
@@ -88,6 +89,13 @@ export function useEvents(apiRef: GridApiRef): void {
   useGridApiOptionHandler(apiRef, GRID_COMPONENT_ERROR, options.onError);
   useGridApiOptionHandler(apiRef, GRID_STATE_CHANGE, options.onStateChange);
 
+  useNativeEventListener(
+    apiRef,
+    apiRef.current.rootElementRef!,
+    GRID_KEYDOWN,
+    getHandler(GRID_KEYDOWN),
+  );
+
   React.useEffect(() => {
     if (apiRef.current.rootElementRef?.current) {
```

Comment:
This effect doesn't work. When it runs, `apiRef.current.rootElementRef?.current` is null. It's already to be removed by #1862, so a native event listener is better.